config/main.py

```diff
@@ -3227,18 +3232,23 @@ def reload(ctx, no_dynamic_buffer, no_delay, dry_run, json_data, ports, verbose)
                     '-t', '{},{}'.format(qos_template_file, qos_fname),
                     '-y', sonic_version_file
                 ]
-                clicommon.run_command(command, display_cmd=True)
+                out, rc = clicommon.run_command(command, display_cmd=True, return_cmd=True)
+                if rc != 0:
+                    click.echo("Command failed due to rc value!")
```

Comment:
this output is not informative to the user

Reply:
Agree, will update it with: click.echo(f"Command failed: {' '.join(command)} with return code: {rc}")

Reply:
Updated in new patch, with click.echo("Some keys in APPL_DB didn't update successfully, please reload configuration.")